Trading/Exchange/bybit/bybit_exchange.py

```diff
@@ -90,27 +94,11 @@ def get_default_type(self):
     async def get_positions(self) -> list:
         params = {}
         raw_positions = []
-        for position_type in ("linear", "inverse"):
-            params["subType"] = position_type
+        for settleCoin in ("USDT", "BTC"):
+            params["settleCoin"] = settleCoin
```

Comment:
:open_mouth:

Reply:
bybit is spliting positions with settlement assets